app/models/special_route.rb

```diff
@@ -0,0 +1,127 @@
+class SpecialRoute
+  def self.publish_all(publishing_api)
+    ROUTES.each do |route|
+      route_type = route.delete(:type) || "exact"
+      route_spec = defaults.merge(route).merge(
+        routes: [
+          path: route[:base_path],
+          type: route_type,
+        ]
+      )
+
+      self.new(route_spec, publishing_api).publish
+    end
+  end
+
+  def initialize(route_spec, publishing_api)
+    @route_spec = route_spec
+    @publishing_api = publishing_api
+  end
+  attr_reader :route_spec, :publishing_api
+
+  def publish
+    publishing_api.put_content_item(route_spec[:base_path], route_spec)
+  end
+
+private
+
+  def self.defaults
+    {
+      format: "special_route",
+      publishing_app: "static",
+      rendering_app: "static",
+      update_type: "major",
+      public_updated_at: Time.zone.now.iso8601,
+    }
+  end
+
+  ROUTES = [
+    {
+      base_path: "/favicon.ico",
+      content_id: "5fc8b4bc-f899-4d4e-ae10-f9beeb172f50",
+      title: "Favicon",
+      description: "The favicon is the image displayed in locations such as the browser tabs.",
+    },
+    {
+      base_path: "/robots.txt",
+      content_id: "c5dad62e-c21a-4fe4-841b-7bcb8a901e52",
+      title: "robots.txt",
+      description: "Provides rules for which parts of GOV.UK are permitted to be crawled by different bots.",
+    },
+    {
+      base_path: "/humans.txt",
+      content_id: "6e92af59-3a73-4db6-b58b-740a02b229d0",
+      title: "humans.txt",
+      description: "In opposition to robots.txt, humans.txt provides information about GOV.UK to interested readers,
+                    such as developers interested in joining GDS.",
+    },
+    {
+      base_path: "/fonts",
+      content_id: "f0115875-227e-4441-a061-3fdc6d79b7e6",
+      title: "Fonts prefix",
+      description: "This location previously held fonts for GOV.UK, but has since been removed.",
+      type: "prefix",
+      rendering_app: "gone",
+    },
```

Comment:
I'm not sure it's worth including this in here.  It was never actually live on the www.gov.uk domain, and was originally included by mistake (the fonts are all served through the assets domain).